monitoring/google/cloud/monitoring_v3/_dataframe.py

```diff
@@ -15,6 +15,12 @@
 """Time series as :mod:`pandas` dataframes."""
 
 import itertools
+try:
+    import pandas
```

Comment:
The extras requires a min version of 17.1. Do we need to do that check here as well?

Reply:
I don't think so.